spl/src/metadata.rs

```diff
@@ -483,7 +473,7 @@ pub fn utilize<'info>(
         token_account: *ctx.accounts.token_account.key,
         token_program: spl_token::ID,
         use_authority: *ctx.accounts.use_authority.key,
-        use_authority_record: None,
```

Comment:
Was this set to `None` intentionally?

Reply:
It was, but I can see now that this is not correct. We need to set the `use_authority_record_pda` as a remaining account.

Reply:
Your change was all we needed to do, so it is all good with this one.

Reply:
Awesome, thanks!